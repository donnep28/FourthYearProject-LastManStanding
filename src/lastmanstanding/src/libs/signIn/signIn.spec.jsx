import React from 'react';
import { shallow } from 'enzyme';
import SignIn from './signIn';

describe('SignIn', () => {
    const props = {
        username: 'username',
        password: 'password',
    };
    describe('Snapshot', () => {
        it('renders', () => {
          const component = shallow(
            <SignIn {...props}/>,
          );
          expect(component).toMatchSnapshot();
        });
    });
});