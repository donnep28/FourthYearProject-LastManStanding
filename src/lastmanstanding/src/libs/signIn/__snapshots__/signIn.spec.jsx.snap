// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn Snapshot renders 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Container))
    component="main"
    maxWidth="xs"
  >
    <div
      className="makeStyles-paper-1"
    >
      <WithStyles(ForwardRef(Avatar))
        className="makeStyles-avatar-2"
      >
        <Memo(ForwardRef(LockOutlinedIcon)) />
      </WithStyles(ForwardRef(Avatar))>
      <WithStyles(ForwardRef(Typography))
        component="h1"
        variant="h5"
      >
        Sign in
      </WithStyles(ForwardRef(Typography))>
      <form
        className="makeStyles-form-3"
        noValidate={true}
      >
        <WithStyles(ForwardRef(TextField))
          autoComplete="uname"
          autoFocus={true}
          fullWidth={true}
          id="username"
          label="Username"
          margin="normal"
          name="username"
          onChange={[Function]}
          required={true}
          variant="outlined"
        />
        <WithStyles(ForwardRef(TextField))
          autoComplete="current-password"
          fullWidth={true}
          id="password"
          label="Password"
          margin="normal"
          name="password"
          onChange={[Function]}
          required={true}
          type="password"
          variant="outlined"
        />
        <WithStyles(ForwardRef(FormControlLabel))
          control={
            <ForwardRef(WithStyles)
              color="primary"
              value="remember"
            />
          }
          label="Remember me"
        />
        <WithStyles(ForwardRef(Button))
          className="makeStyles-submit-4"
          color="primary"
          fullWidth={true}
          onClick={[Function]}
          type="submit"
          variant="contained"
        >
          Sign In
        </WithStyles(ForwardRef(Button))>
        <div>
          <Link
            onClick={[Function]}
            to="/forgotPassword"
          >
            Forgot Password?
          </Link>
        </div>
      </form>
    </div>
  </WithStyles(ForwardRef(Container))>
</Fragment>
`;
